azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestUtil.java

```diff
@@ -253,9 +256,10 @@ private FlowRunner createFromExecutableFlow(final EventCollectorListener eventCo
     }
     exFlow.getExecutionOptions().addAllFlowParameters(flowParams);
     this.executorLoader.uploadExecutableFlow(exFlow);
+    final CommonMetrics commonMetrics = new CommonMetrics(new MetricsManager(new MetricRegistry()));
     final FlowRunner runner =
         new FlowRunner(exFlow, this.executorLoader, this.projectLoader,
-            this.jobtypeManager, azkabanProps, null, mock(AlerterHolder.class));
+            this.jobtypeManager, azkabanProps, null, mock(AlerterHolder.class), commonMetrics);
```

Comment:
Do we expect certain metrics get populated in commonMetrics?
If yes, can we assert expected vs real values?

Reply:
No metrics will be populated by default. We can add asserts when the test starts to set certain metrics.